app/src/main/java/com/antosik/benchproject/app/common/constants/Constants.kt

```diff
@@ -0,0 +1,8 @@
+package com.antosik.benchproject.app.common.constants
+
+internal class Constants {
+
+    companion object {
+        const val GITHUB_URL = "https://github.com/Nataniel-Antosik"
```

Comment:
You could also use BuildConfig for such purposes

Reply:
It is in `MoreViewModel.kt`.